locust/runners.py

```diff
@@ -299,25 +305,23 @@ def start(self, user_count: int, spawn_rate: float, wait: bool = False):
             if self.environment.host is not None:
                 user_class.host = self.environment.host
 
-        self.target_user_classes_count = weight_users(self.user_classes, user_count)
-
-        local_worker_node = WorkerNode(id="local")
-        local_worker_node.user_classes_count = self.user_classes_count
-
         if self.state != STATE_INIT and self.state != STATE_STOPPED:
             self.update_state(STATE_SPAWNING)
 
+        if self._users_dispatcher is None:
+            self._users_dispatcher = UsersDispatcher(
+                worker_nodes=[self._local_worker_node], user_classes=self.user_classes
+            )
+
         logger.info("Ramping to %d users using a %.2f spawn rate" % (user_count, spawn_rate))
```

Comment:
Maybe rephrase this to "Ramping to %d users at a rate of %.1f per second"? (particularly the "per second" makes the sentence make more sense)

Reply:
Done (in my other PR)